_docs/_latest/develop/test/run-website-locally/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Build the Website
+order: 046
+hide:
+  right_sidebar: true
+---
+
+# {{ page.title }}
+
+Use our guides on how best to set up your development environment, make changes,
+and then test those changes.
+
+**[Run website server locally]({% link _docs/latest/develop/test/run-website-locally/website.md %})**
```

Comment:
This link does not work. I don't think this website content belongs in the testing section.

Reply:
I had moved instructions to run website locally to `testing` section as this section has instructions to run and test unit and integration tests locally. Website content has been moved out of this section.